### ISSUE TITLE
Spike - Applicability of RIO within ecosystem

### ISSUE THREAD

**Issue by MarcelUntied** [stichtingsem/pilot-phase#12]

**Comment by MarcelUntied** [comment]
Niet meenemen in de pilot wel voor vervolgstappen

**Comment by eliashassing154** [comment]
Propose to close.

**Comment by MarcelUntied** [comment]
@eliashassing154 I don't agree. I adjusted my earlier remark. technical realisation not in scope. Research in scope

**Comment by eliashassing154** [comment]
But wouldn't it make more sense to treat that as a separate track @MarcelUntied? Because the pilot teams won't be doing the research (I think)?

**Comment by MarcelUntied** [comment]
@eliashassing154 That is ok. Maybe the project team could do the research?

**Comment by mcginkel** [comment]
We can add the RIO-onderwijsAanbieder code to the School definition, next to the existing ID's.

**Comment by MarcelUntied** [comment]
[OSR verslag 22 February](https://github.com/stichtingsem/pilot-phase/blob/main/documents/osr/20210218%20Gespreksverslag%20vervolgsessie%20OSR.pptx)

**Comment by MarcelUntied** [comment]
[Report RIO 22 February](https://github.com/stichtingsem/pilot-phase/blob/main/documents/rio/20210222%20Gespreksverslag%20vervolgsessie%20RIO.pptx)
